tests/url_reader.prm

```diff
@@ -0,0 +1,102 @@
+# A model using the ascii data plugin to read in a file
+# generated by geomIO. 
+# See the manual for more information.
+
+set Output directory                       = output-geomIO
+set Dimension                              = 2
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+
+set Adiabatic surface temperature          = 1613.0
+
+subsection Geometry model
+  set Model name = box
+
+# The extents of the box in m is the same as the width
+# and height of the drawing in px 
+# (an A4 page = 7350x10500 px).
+  subsection Box
+    set X extent = 7350
+    set Y extent = 10500
+  end
+end
+
+subsection Initial temperature model
+  set Model name = adiabatic
+  subsection Adiabatic
+    set Amplitude = 300
+    set Radius    = 250000
+  end
+end
+
+# We need one compositional field that will be assigned 
+# the values read in from the ascii data plugin.
+subsection Compositional fields
+  set Number of fields = 1
+end
+
+# We use the ascii data plugin to read in the file created with geomIO.
+subsection Initial composition model
+  set Model name = ascii data
+
+  subsection Ascii data model
+    set Data directory       = 
+    set Data file name       = http://localhost:8080/opendap/data/csv/geomIO_urlTest.csv
```

Comment:
how is this going to work without a local http server?

Reply:
We can use a file:// URL to local test data.

Reply:
perfect. You can use ``$ASPECT_SOURCE_DIR``, which will be expanded, see https://github.com/geodynamics/aspect/blob/master/tests/ascii_data_adiabatic_conditions.prm#L13 for example